src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -33,7 +37,7 @@ public partial class BrotliStream : Stream
         Decoder _decoder;
         Encoder _encoder;
         public BrotliStream(Stream baseStream, CompressionMode mode, bool leaveOpen, int BuffSize, uint windowSize, uint quality) : this(baseStream, mode, leaveOpen, BuffSize) {
```

Comment:
Please format this code with open brace on new line and possibly newline + indent for calling the other constructor.